foundation-extensions/experience-platform-extension/README.md

```diff
@@ -31,82 +38,85 @@ The Edge configuration used by the client-side implementation is one of the foll
 
 {% tabs %}
 {% tab title="Android" %}
-### Java
+#### Java
 
 1. Add the Mobile Core and Edge extensions to your project using the app's Gradle file.
 
    ```java
-   implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
+   implementation 'com.adobe.marketing.mobile:core:1.+'
    implementation 'com.adobe.marketing.mobile:edge:1.+'
+   implementation 'com.adobe.marketing.mobile:edgeidentity:1.+'
    ```
 
 2. Import the Mobile Core and Edge extensions in your application class.
 
    ```java
     import com.adobe.marketing.mobile.*;
    ```
+
 {% endtab %}
 
-{% tab title="iOS — Swift" %}
+{% tab title="iOS (AEP 3.x)" %}
+
 1. Add the Mobile Core and Edge extensions to your project using Cocoapods. Add following pods in your `Podfile`:
 
    ```swift
    use_frameworks!
    target 'YourTargetApp' do
-       pod 'AEPCore'
-       pod 'AEPLifecycle'
-       pod 'AEPIdentity'
-       pod 'AEPSignal'
-       pod 'AEPEdge'
+   	pod 'AEPCore'
+   	pod 'AEPEdge'
+   	pod 'AEPEdgeIdentity'
    end
    ```
-
+   
 2. Import the Mobile Core and Edge libraries:
 
-### Swift
+#### Swift
 
 ```swift
 // AppDelegate.swift
 import AEPCore
 import AEPEdge
-import AEPIdentity
-import AEPLifecycle
-import AEPSignal
+import AEPEdgeIdentity
 ```
 
-### Objective-C
+#### Objective-C
 
-```text
+```objective-c
 // AppDelegate.h
 @import AEPCore;
 @import AEPEdge;
-@import AEPIdentity;
-@import AEPLifecycle;
-@import AEPSignal;
+@import AEPEdgeIdentity;
 ```
 {% endtab %}
+
+{% tab title="iOS (ACP 2.x)" %}
+
+This extension is built based on the AEPCore, please follow [the guide for migrating to Swift](https://aep-sdks.gitbook.io/docs/resources/migrate-to-swift).
```

Comment:
Maybe reword to:
"This extension is built on the AEPCore (3.x) and cannot be used with the ACPCore (2.x). Please follow the guide for migrating to the Swift AEPCore."